### DISCUSSION TITLE
Point (Marker) is going to end when dblclicked

### DISCUSSION THREAD

**Discussion by seperman-jh** [bbc/peaks.js#509] [Q&A] [answered]
Hello Chris,

I'm super thankful for you creating peaks.js. It's been a real game-changer for me, and I'm loving it!
I'm currently working on my app, and I recently upgraded to 3.1.0, and I've encountered one issue. (not happend in 3.0.0-beta10)

1. In the zoom view, when I double-click, the pointer forcibly moves to the very end.
- I want the pointer to stay in place when I double-click, but suddenly, it moves backward.
- I tried using preventViewEvent() since it was added in the higher versions, but the issue remains the same.

I've been continuing my research, but I couldn't find a solution, so I'm leaving this question here.
Please check this below.

https://github.com/bbc/peaks.js/assets/13728223/f78ea9d8-6ca7-4b54-bc05-4718eb16079f

if not working, please click another video link: 
https://drive.google.com/file/d/11zgWf5yNQ68Ey8VqMdnX2uaPzZHfQfiC/view?usp=sharing

`currentPeaksInstance?.on('zoomview.dblclick', (event) => {
  event.evt.preventDefault();
  setMsg('zoomview dblClicked!!');
  setCount(count + 1);
});`

I'm always thankful for your feedback and continuous updates.
Thank you

**Comment by chrisn** [comment]
> I'm super thankful for you creating peaks.js. It's been a real game-changer for me, and I'm loving it!

Thank you for saying this, it means a lot.

> In the zoom view, when I double-click, the pointer forcibly moves to the very end.

This is definitely a bug, introduced in https://github.com/bbc/peaks.js/commit/0eca0be0bf00c54f80294b005a23c7e7a10ea296. I think that v3.0.0-beta.12 is OK.

I'll work on a fix...

**Comment by chrisn** [comment, marked as answer]
I have just released v3.2.0, which should fix this.

**Reply by seperman-jh** [reply]
Wow, thanks so much for the quick fix!
I tested it a little while ago, and it works perfectly now. 
I really appreciate it.
